modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/util/MuleExtensionUtils.java

```diff
@@ -464,9 +465,9 @@ public static ExtensionModel loadExtension(Class<?> clazz, Map<String, Object> p
     return new DefaultJavaExtensionModelLoader().loadExtensionModel(clazz.getClassLoader(), dslResolvingContext, params);
   }
 
-  public static String getImplicitConfigurationProviderName(ExtensionModel extensionModel,
+  public static String getImplicitConfigurationProviderName(MuleContext muleContext, ExtensionModel extensionModel,
```

Comment:
get the id (better called `prefix`) directly instead of passing the `MuleContext` around